internal/processor/printer.go

```diff
@@ -105,6 +106,12 @@ func (p *SDCPrinter) ReProcess(pfr *pkg.PrintFileRequest) error {
 	// increment the number of attempts
 	numberOfAttempts := pfr.Attempts
 	pfr.Attempts = numberOfAttempts + 1
+	// log an error for each retry
+	if pfr.Attempts > 1 {
+		strAttempts := strconv.Itoa(pfr.Attempts)
+		retryFile := pfr.DataFilename
+		logger.Error("Retried connection " + strAttempts +" times for file " + retryFile)
```

Comment:
This should use the zap structured logging rather than building up a string
i.e.
```		
logger.Error("Retried connection", zap.Int("attempts", attempts), zap.String("file " + retryFile))
```

Reply:
Good idea thanks